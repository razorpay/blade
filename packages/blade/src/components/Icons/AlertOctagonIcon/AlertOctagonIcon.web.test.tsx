import AlertOctagonIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<AlertOctagonIcon />', () => {
  it('should render AlertOctagonIcon', () => {
    const { container } = renderWithTheme(
      <AlertOctagonIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
