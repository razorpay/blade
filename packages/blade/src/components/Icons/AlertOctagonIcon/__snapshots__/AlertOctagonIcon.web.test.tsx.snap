// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertIcon /> should render AlertIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    fill="none"
    height="20px"
    viewBox="0 0 20 20"
    width="20px"
  >
    <path
      d="M10.0002 5.83331C10.4604 5.83331 10.8335 6.20641 10.8335 6.66665V9.99998C10.8335 10.4602 10.4604 10.8333 10.0002 10.8333C9.53992 10.8333 9.16683 10.4602 9.16683 9.99998V6.66665C9.16683 6.20641 9.53992 5.83331 10.0002 5.83331Z"
      fill="hsla(216, 27%, 36%, 1)"
    />
    <path
      d="M10.0002 14.1666C10.4604 14.1666 10.8335 13.7935 10.8335 13.3333C10.8335 12.8731 10.4604 12.5 10.0002 12.5C9.53992 12.5 9.16683 12.8731 9.16683 13.3333C9.16683 13.7935 9.53992 14.1666 10.0002 14.1666Z"
      fill="hsla(216, 27%, 36%, 1)"
    />
    <path
      clip-rule="evenodd"
      d="M5.96091 1.07739C6.11171 0.926587 6.32004 0.833313 6.55016 0.833313H13.4502C13.6712 0.833313 13.8831 0.92111 14.0394 1.07739L18.9228 5.96072C19.079 6.117 19.1668 6.32897 19.1668 6.54998V13.45C19.1668 13.671 19.079 13.883 18.9228 14.0392L14.0394 18.9226C13.8831 19.0788 13.6712 19.1666 13.4502 19.1666H6.55016C6.32915 19.1666 6.11719 19.0788 5.96091 18.9226L1.07757 14.0392C0.921294 13.883 0.833496 13.671 0.833496 13.45V6.54998C0.833496 6.32897 0.921293 6.117 1.07757 5.96072L5.96091 1.07739ZM6.89534 2.49998L2.50016 6.89516V13.1048L6.89534 17.5H13.105L17.5002 13.1048V6.89516L13.105 2.49998H6.89534Z"
      fill="hsla(216, 27%, 36%, 1)"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
